app/cells/folio/console/form/errors/errors.js

```diff
@@ -1,76 +1,75 @@
 
-      key = key.replace('_attributes', '').replace(/\]\[\d*\]\[/, '.')
-
-      let found = false
-
-      this.buttonTargets.forEach((buttonTarget) => {
-        if (found) return
-        if (!buttonTarget.classList.contains('f-c-form-errors__button--hidden')) return
-        if (buttonTarget.dataset.errorField !== key) return
-        buttonTarget.classList.remove('f-c-form-errors__button--hidden')
-        buttonTarget.formGroup = formGroup
-        found = true
-      })
+            this.buttonTargets.forEach((buttonTarget) => {
+                if (found) return
+                if (!buttonTarget.classList.contains('f-c-form-errors__button--hidden')) return
+                if (buttonTarget.dataset.errorField !== key) return
+                buttonTarget.classList.remove('f-c-form-errors__button--hidden')
+                buttonTarget.formGroup = formGroup
+                found = true
+            })
+        }
     }
-  }
-
-  disconnect () {
-    this.buttonTargets.forEach((buttonTarget) => {
-      delete buttonTarget.formGroup
-    })
-  }
 
-  onButtonClick (e) {
-    e.preventDefault()
+    disconnect() {
+        this.buttonTargets.forEach((buttonTarget) => {
+            delete buttonTarget.formGroup
+        })
+    }
 
-    const btn = e.currentTarget
+    onButtonClick(e) {
+        e.preventDefault()
 
-    if (!btn.formGroup) return
+        const btn = e.currentTarget
+        console.log(btn.formGroup)
+        if (!btn.formGroup) return
 
-    const tab = btn.formGroup.closest('.tab-pane')
+        const tab = btn.formGroup.closest('.tab-pane')
 
-    if (tab && !tab.classList.contains('active')) {
-      for (const tabLink of document.querySelectorAll('.nav-tabs .nav-link')) {
-        if (tabLink.href.split('#').pop() === tab.id) {
-          tabLink.click()
-          break
+        if (tab && !tab.classList.contains('active')) {
+            for (const tabLink of document.querySelectorAll('.nav-tabs .nav-link')) {
+                if (tabLink.href.split('#').pop() === tab.id) {
+                    tabLink.click()
+                    break
+                }
+            }
         }
-      }
-    }
 
-    btn.formGroup.scrollIntoView({ behavior: 'smooth', block: 'center' })
+        btn.formGroup.scrollIntoView({behavior: 'smooth', block: 'center'})
 
-    if (btn.formGroup.dataset.controller) {
-      btn.formGroup.dataset.controller += ' f-c-danger-box-shadow-blink'
-    } else {
-      btn.formGroup.dataset.controller = 'f-c-danger-box-shadow-blink'
+        if (btn.formGroup.dataset.controller) {
+            btn.formGroup.dataset.controller += ' f-c-danger-box-shadow-blink'
+        } else {
+            btn.formGroup.dataset.controller = 'f-c-danger-box-shadow-blink'
+        }
+        const input = btn.formGroup.querySelector('.form-control')
+        if (input) input.focus()
     }
-
-    const input = btn.formGroup.querySelector('.form-control')
-    if (input) input.focus()
-  }
 })
```

Comment:
Proc je to preindentovane? Pro javascript pouzivame 2 mezery, tak to prosim vrat.